src/esp/gfx/CubeMap.cpp

```diff
@@ -155,6 +155,7 @@ void CubeMap::recreateTexture() {
   // color texture
   if (flags_ & Flag::ColorTexture) {
     auto& colorTexture = texture(TextureType::Color);
```

Comment:
Should this line be removed or the line above it be deleted? It seems weird to overwrite colorTexture and depthTexture immediately like this.

Reply:
> overwrite

Yes, it overrides whatever existing in the variable. This function is called `recreateTexture()`. It is its job.
The function `texture` is to make sure the type is legal, and return the texture container properly.